modules/custom/d_content/config/update/d_content.8139.yml

```diff
@@ -0,0 +1,33 @@
+field.field.node.content_page.field_page_section:
```

Comment:
This config should be inside d_mautic module

Reply:
@grzegorz-pietrzak-droptica fixed :)